### PR TITLE
add koda-inc.md

### PR DESCRIPTION
**Pull request by koda9-inc**

### CODE REVIEW

**Comment by dineshshenoy** [comment]
I couldn't locate your form entry (https://slingshot.filecoin.io/register-now).  Could you submit this form to complete the registration process?

**Comment by cwaring** [comment]
@koda9-inc following up on this; your registration form is required for this entry to qualify https://slingshot.filecoin.io/register-now

**Review by GregTheGreek — APPROVED** [review]

**Comment by koda9-inc** [comment]
@dineshshenoy We have submitted the form, and here are the code and Id: 

The verification code is:

barman-distort-speculate

(Submission ID: 672993312)

**Review by dineshshenoy — APPROVED** [review]

File: participants/koda-inc.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+## (8) What pre-processing are you doing before ingesting the data?
+
+•	The data uploaded by the user will be transferred to the server through the API interface and then to the IPFS node. The data will be divided into multiple smaller pieces when the file size exceeds a certain specification. 
+•	Certain database will has been used to store the data. The data table will includes File name, File ID, File size, File type, and File CID list (when a file is divided into multiple pieces)
+
+
+## (9)  What tech stack will you use to store the data?
+
+Lotus
```
Comment (dineshshenoy) [inline participants/koda-inc.md:53]: You may want to consider Powergate or IPFS for hot storage retrieval.

**Review by dineshshenoy — APPROVED** [review]

**Pull request merged**